public_app/App.js

```diff
@@ -0,0 +1,45 @@
+var React = require('react');
+var ReactDOM = require('react-dom');
+var ReactRouter = require('react-router-dom');
+var Router = ReactRouter.BrowserRouter;
+var Route = ReactRouter.Route;
+var Switch = ReactRouter.Switch;
+var Link = require ('react-router-dom').Link;
+import Nav              from './components/Nav';
+import AllMovies        from './pages/AllMovies';
+import Home             from './pages/Home';
+import MoviesDetails    from './pages/MoviesDetails';
+import Register         from './pages/Register';
```

Comment:
You either align things like `from` or you don't, don't do something in the middle that looks awful https://shkspr.mobi/blog/2014/11/why-i-vertically-align-my-code-and-you-should-too/